notebooks/.ipynb_checkpoints/portfolio_prices-checkpoint.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
Look how bad a ipynb is managed by github. I don't think it's necessary. If the idea is to show that it works, adding a docstring should to the same. So I think we should remove the ipynb and document the functions.

But if you want to keep the ipynb, at least add a gitignore to your local repository to avoid commiting .ipynb_checkpoints